test/test_modem.py

```diff
@@ -207,6 +207,8 @@ def writeCallbackFunc(data):
                  (['FGH,RTY,UIO\r\n', 'OK\r\n'], ['FGH', 'RTY', 'UIO']), # nasty, but possible
                  # ZTE-like response: do not start with +CLAC, and use multiple lines
                  (['A\r\n', 'BCD\r\n', 'EFGH\r\n', 'OK\r\n'], ['A', 'BCD', 'EFGH']),
+                 # Siemens response: like ZTE but each command has AT prefix
+                 (['AT&F\r\n', 'AT&V\r\n', 'AT&W\r\n', 'AT+CACM\r\n', 'OK\r\n'], ['&F', '&V', '&W', '+CACM']),
```

Comment:
Does Siemens modem actually send `&` characters?

Reply:
AT& commands exist.

AT&F = restore factory defaults.

[all & commands here](https://en.wikibooks.org/wiki/Serial_Programming/Modems_and_AT_Commands/%26_Commands)

Reply:
Oh you probably meant '&F', '&V', '&W' part. Let's wait for @ffix

Reply:
Thank you for a reminder! I just got suspicious, because @ffix have not mentioned those commands inside PR description. Sorry for the confusion.

Reply:
> Does Siemens modem actually send `&` characters?
It's my mistake. It is actually Teleofis modem, not Siemens.
I've updated description.

Reply:
> Thank you for a reminder! I just got suspicious, because @ffix have not mentioned those commands inside PR description. Sorry for the confusion.

You're right. I copied these commands from modem response directly.